fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -94,7 +94,15 @@
       const url = this.href;
       http('HEAD', url, function(res) {
         if (res.status >= 400 || res.getResponseHeader('Link') == null) {
-          location.href = url;
+          var newLocation = url;
```

Comment:
The fetch API _would_ be perfect for this, unless we care about supporting Safari and IE. One option would be to polyfill `fetch` with `XMLHttpRequest` for those browsers, but doing so might make this code a lot more complicated than it needs to be, especially for a feature that has nothing to do with the API spec.